common/src/main/include/SensoredTransmission.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <frc/SpeedController.h>
+#include <sensors/Encoder.h>
+
+/* Combines the encoder and motor parts into one data structure,
+so that both Spark + encoder and Talon SRX can be treated the same. */
+
+namespace curtinfrc {
```

Comment:
One more change - when adding documentation, we usually do it like so:

```cpp
/**
 * This is a brief description
 * 
 * This is a much longer description
 */
class MyClass {
 public:

  /**
   * A variable, contains some value
   */
   double myVar;

  /**
   * Does a thing
   *
   * @param myParam Some parameter
   * @returns Some value
   */
  double myFunction(double myParam);
};
```

For common, we should try to document as much as possible. See `talon_srx` for reference.
This makes it pop up correctly in VSCode / when generating docs